src/PharoThings-Devices-LCD1602.package/PotLCD1602Device.class/instance/initialize.st

```diff
@@ -0,0 +1,10 @@
+accessing
+initialize
+	gpioArray := Array 
+		with: (PotClockGPIOPin id: 27 number: 2) "gpio2 RS - set character '1' ou lcd commands '0' " 
+		with: (PotClockGPIOPin id: 22 number: 3) "gpio3 EN - clock enable " 
+		with: (PotClockGPIOPin id: 25 number: 6) "gpio6 D4 - data bit" 
+		with: (PotClockGPIOPin id: 24 number: 5) "gpio5 D5 - data bit"
+		with: (PotClockGPIOPin id: 23 number: 4) "gpio4 D6 - data bit"
+		with: (PotClockGPIOPin id: 18 number: 1)."gpio1 D7 - data bit"
+	backlight := (PotPwmGPIOPin id: 12 number: 26)."gpio26 BL - back light PWM"
```

Comment:
Problem with this method is that you can't reuse your device with another board. Because peripheral pins are hardcoded here. 
Peripheral pins should be passed to device when it is created. It is like in real word you need to plug device to correct pins.
But I understand the reason why you wrote it like this. If device requires complex configuration we would like to incapsulate it in some place and reuse it. 
Then proper solution would be to double dispatch logic through board instance because pins required for device depend on concrete board scheme. 
It can be done in #connect method: 
```Smalltalk
LCD1602Device>>connect
   self isConfigured ifFalse: [  board configureLCD1602Device: self]
   ....
LCD1602Device>>isConfigured
   "Just as possible solution"
   ^clockPin notNil
```
And new board class will implement it like:
```Smalltalk
RpiBoard3B>>configureLCD1602Device: aDevice
   aDevice
      modePin: (self pinWithId: 27);
      clockPin: (self pinWithId: 22);
      dataPins: (#(25 24 23 18) collect: [:id | self pinWithId: id]);
      backlight: (self pinWithId: 12). 
```
Other type of boards will implement it differently by specifying own pins suitable for this device.
Notice that here I also suggest to split gpioArray to more logical set of pin variables. It will make code of users much cleaner. Look at other comments